src/classes/BGE_DataImportBatchEntry_CTRL.cls

```diff
@@ -88,58 +88,63 @@ public with sharing class BGE_DataImportBatchEntry_CTRL {
         }
 
         DataImportModel dataImportModel = new DataImportModel();
-        Integer rowOffset = 0;
-
         dataImportModel.id = batchId;
-        dataImportModel.dataImportRows = getDataImportRows(batchId, rowOffset);
+        dataImportModel.dataImportRows = getDataImportRows(batchId, 0);
         dataImportModel.setColumns();
         dataImportModel.labels = getBatchDataImportLabels();
         dataImportModel.isNamespaced = String.isNotBlank(UTIL_Namespace.getNamespace());
-        dataImportModel.totalCountOfRows = getDataImportBatchTotalCount(batchId);
+        dataImportModel = getDataImportBatchTotals(dataImportModel);
```

Comment:
I know we discussed maybe having an new separate method for the controller to use to get the batch total for this work item.  If we are going to go this route and modify the model that is used by current _and_ old BGE, then we'll have to make sure we test old BGE also.